dds_web/api/schemas/user_schemas.py

```diff
@@ -219,6 +219,12 @@ def make_user(self, data, **kwargs):
         # Verify and transfer invite keys to the new user
         if verify_and_transfer_invite_to_user(token, new_user, data.get("password")):
             for project_invite_key in invite.project_invite_keys:
+                if isinstance(new_user, models.ResearchUser):
+                    project_user = models.ProjectUsers(
+                        project_id=project_invite_key.project_id, owner=project_invite_key.owner
+                    )
```

Comment:
I don't know how you have this working. You have written a similar line like this before that I tried to use and it didn't work because for example now you are trying to create ProjectUsers object which is missing the primary key user_id. That just didn't work for me and honestly I wouldn't expect it to work. Just sharing my observation.

Reply:
That's very strange that it's not working..
The `project_user` is added in the line 226 below, that's how it's working. At least for me I get a row in the ProjectUser table